blocks/left-rail/left-rail.js

```diff
@@ -40,10 +52,13 @@ export default async function decorate(block) {
   }
 
   if (embed) {
-    const fragmentBlock = buildBlock('fragment', `<a href='${embed}'/>`);
+    const fragmentBlock = buildBlock('fragment', `<a href='${embed}' style='display:none'/>`);
```

Comment:
shouldn't we be creating <a> tag with a class identifier and add the "display:none" in corresponding CSS file. My experience with adding styling in JS is not so good and while it is not conventionally wrong but it makes debugging a little complex when it comes to identify the source of styling at first place. Although trivial but i'd leave this to call :)

Reply:
I'm not sure it's needed in this case but you have a point -- this could prevent a layout shift!  I'll incorporate your suggestion.  Thanks!